File: Seapal_play/app/controllers/Map.java
package controllers;

import play.*;
import play.mvc.*;

import views.html.*;
import views.html._include.*;
import views.html._include.map.*;

public class Map extends Controller {
  
  public static Result index() {
    return ok(map.render(header.render(),
    						header_app.render(),
    						header_app_map.render(),
    						navigation.render("app_map"),
    						navigation_app.render("app_map"),
    						map_content.render(),
    						weather_chart.render()));
  }
  
}

File: seapal_play2.1/app/controllers/Map.scala
package controllers

import play.api._
import play.api.mvc._

import views.html._
import views.html._include._
import views.html._include.mapInc._

object Map extends Controller {

  def index = Action {	    
	  Ok(map(header_app.render(),
	  			header_app_map.render(),
	  			navigation.render("app_map"),
	  			navigation_app.render("app_map"),
	  			map_content.render(),
	  			weather_chart.render()))
  }
}